src/Http/Livewire/LivewireDatatable.php

```diff
@@ -29,11 +30,11 @@ class LivewireDatatable extends Component
     use WithPagination, WithCallbacks, WithPresetDateFilters, WithPresetTimeFilters;
```

Comment:
@CircleLinkHealth/developers Thanks for the reviews, just making sure you also checked this file since it is hidden cause is too big